rfcs/0175-appeals-council.md

```diff
@@ -0,0 +1,170 @@
+
+This proposal seeks to establish clear foundations for a shared understanding of moderation practices through the *Memorandum of Understanding on Equitable Moderation* and to ensure these foundations are upheld in practice. It aims to prevent the NixOS foundation mission statement from being misused to enforce a homogenizing moderation policy.
+
+The *Memorandum* should be seen as a normatively binding interpretation of the moderation team's mission statement.
+
+This document aims to articulate an overlapping consensus on reasonable expectations for anyone participating as an equal member in this community, emphasizing that community members are accountable to moderation and vice versa.
+
+# Detailed Design
+[design]: #detailed-design
+
+## Goals
+
+- Adopt the *Memorandum of Understanding on Equitable Moderation* to foster a shared understanding of equitable moderation practices that accommodate a community with diverse sociopolitical views.
+- Ensure transparent accountability of moderation practices to community members.
+- Ground community member accountability in publicly available general principles, providing a stable basis for behavioral expectations.
+- Make wide-ranging decisions regarding community member exclusion understandable.
+- Promote a diverse moderation team to balance diverse perspectives on controversial issues.
+- Establish an *Appeals Council* to hold moderation team members and procedures accountable and to allow for the revision of significant decisions if they fail to meet the community's reasonable standards.
+
+## Memorandum of Understanding on Equitable Moderation
+
+In response to the impression left by past events and with a fervent desire for improvement, we adopt the following principles and maxims as the *Memorandum of Understanding on Equitable Moderation*. These principles serve to clarify the overlapping consensus on reasonable expectations for community participation:
+
+1) We, as moderators, commit to basing decisions on general principles and concrete reasons that apply equally to all community members, irrespective of characteristics such as sociopolitical orientation, economic status, identity, or religion.
+2) We adhere to the principles of *equality* and *objectivity*. Equality refers to the status of each community member, while objectivity pertains to the consistent, unbiased reasoning behind our decisions.
+3) A *concrete reason* is a verifiable state of affairs that can be demonstrated to other community members upon request.
+4) We acknowledge that *concrete reasons* are not self-explanatory, except in extreme cases, and we will provide explanations relating them to the general principles guiding our decisions.
```

Comment:
So you know that the previous point is actually unworkable, and yet you proposed it anyway?

Reply:
I don't understand how that is what that says?

Person A says X.  Person A is banned because of saying X(the concrete reason).  Person A asks why they were banned, moderators explain how X violates rules and why it is harmful to the community to allow it.